packages/reporter-dom/src/index.ts

```diff
@@ -9,6 +9,7 @@ import type {
 export interface DomReporterOptions {
   listType?: 'ul' | 'ol';
   single?: boolean;
+  singleClass?: string;
```

Comment:
I like this! I just don't know how I feel about having the property work only for single cases 🤔. 
I feel more attributes might be needed in the future so an API like this might be better:

```javascript
{
  singleAttributes: {
    class: /* ... */,
  },
}
```

But I'm open to suggestions.

Reply:
Also adding this functionality also for the list variant would be really helpful.